packages/generator/src/service/package-json.ts

```diff
@@ -1,16 +1,31 @@
 import { unixEOL } from '@sap-cloud-sdk/util';
 import { packageJsonBase } from '@sap-cloud-sdk/generator-common/internal';
 import type { ODataVersion } from '@sap-cloud-sdk/util';
-import type { PackageJsonOptions as PackageJsonOptionsBase } from '@sap-cloud-sdk/generator-common/internal';
 
 /**
  * @internal
  */
-export interface PackageJsonOptions extends PackageJsonOptionsBase {
+export interface PackageJsonOptions {
```

Comment:
generator-common package is shared between odata and openapi generators. Have we check if this affects openapi generator?